src/NzbDrone.Core.Test/IndexerSearchTests/SearchDefinitionFixture.cs

```diff
@@ -13,7 +13,7 @@ public class SearchDefinitionFixture : CoreTest<SingleEpisodeSearchCriteria>
         [TestCase("Hawaii Five-0", Result = "Hawaii+Five+0")]
         [TestCase("Franklin & Bash", Result = "Franklin+and+Bash")]
         [TestCase("Chicago P.D.", Result = "Chicago+PD")]
-        [TestCase("Kourtney And Khlo� Take The Hamptons", Result = "Kourtney+And+Khloe+Take+The+Hamptons")]
+        [TestCase("Kourtney And Khlo\u00E9 Take The Hamptons", Result = "Kourtney+And+Khloe+Take+The+Hamptons")]
```

Comment:
Depending on which IDE is used. Xamarin Studio and MonoDevelop freaks out due to the unicode character. Using the escaped format works fine for both OSX and Windows development